### PR TITLE
Update to latest package:graphs

### PR DESCRIPTION
**Pull request by natebosch**
Will roll this out package by package instead of using
dependency_overrides to get through CI.

### CODE REVIEW

File: build_runner_core/lib/src/package_graph/apply_builders.dart [hunk 1]
```diff
@@ -268,17 +268,18 @@ Future<List<BuildPhase>> createBuildPhases(
...
             if (!targetGraph.allModules.containsKey(key)) {
               _logger.severe('${node.target.key} declares a dependency on $key '
                   'but it does not exist');
               throw CannotBuildException();
             }
             return targetGraph.allModules[key];
-          })?.where((n) => n != null));
+          })?.where((n) => n != null),
+      equals: (a, b) => a.target.key == b.target.key,
+      hashCode: (node) => node.target.key.hashCode);
```
Comment (kevmoo) [inline build_runner_core/lib/src/package_graph/apply_builders.dart]: nit: try a trailing common at the end of the param list, I think it makes it easier to read

Comment (natebosch) [reply build_runner_core/lib/src/package_graph/apply_builders.dart]: It has almost zero impact here... I don't think it's worth the extra line

**Comment by kevmoo** [comment]
ack

**Comment by natebosch** [comment]
Turns out `_test` would need a bunch of hacks to not upgrade all at once... will do it all in 1 go with overrides.

**Comment by kevmoo** [comment]
ack

**Comment by kevmoo** [comment]
So green travis then publish them all right away?

**Comment by natebosch** [comment]
> So green travis then publish them all right away?

Yeah that's my plan. I'll submit this with a green travis, then submit a change with a red travis that drops the overrides and publish everything.

File: build_runner_core/lib/src/package_graph/apply_builders.dart (hunk 1 above)
Comment (jakemac53) [reply build_runner_core/lib/src/package_graph/apply_builders.dart]: Bobs fears have now been fully realized :sob: :sob: :sob:

**Pull request merged**